bin/schema_generator.sh

```diff
@@ -146,7 +146,10 @@ function main() {
     # Add transpiled BQ schemas
     find . -type f -name "*.schema.json" | while read -r fname; do
         bq_out=${fname/schema.json/bq}
-        jsonschema-transpiler --type bigquery "$fname" > "$bq_out"
+        # Snake case untrustedModules for bug 1565074
+        bq_out=${bq_out//untrustedModules/untrusted_modules}
```

Comment:
While this will work, the double slash has a [different meaning](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) and is probably a typo.

Reply:
My intention here is that double slash means replace all:

> If pattern begins with ‘/’, all matches of pattern are replaced with string. Normally only the first match is replaced

Reply:
Ok this actually brings up an interesting point that this new directory may not exist since we're replacing `untrustedModules` multiple times. Likely we'll need to do something about that.

Reply:
`bq_out` is a path like `telemetry/untrustedModules/untrustedModules.1.bq` and we need to replace both those instances of `untrustedModules` in that string.

Reply:
See edit above about the directory not existing. I think that for the purposes of downstream if only the second part is changed we're good, but I would have to check when I get back. It's a little awkward to have the json schema in untrustedModules and the bq schema in a different directory but as long as it works I'm ok with it.

Reply:
> Ok this actually brings up an interesting point that this new directory may not exist since we're replacing untrustedModules multiple times. Likely we'll need to do something about that.

In the last force-push, I added a mkdir call on the next line that should handle that.

> additionally we use by convention - in docTypes, so changing this to _ may have effects if the downstream jenkins script is assuming the directory name and first part of the file name match

Good point. This should be `bq_out=${bq_out//untrustedModules/untrusted-modules}`. I think if I make that change, we should be good to go without changing the jenkins logic. We aren't doing the full snake casing here; we're only handling untrustedModules as a special case.